app/controllers/resources_controller.rb

```diff
@@ -1,17 +1,36 @@
 class ResourcesController < ApplicationController
   def index
     category_id = params[:category_id]
-    resources = if category_id
-                  Resource.joins(:categories).where('categories.id' => category_id)
-                else
-                  Resource.all
-                end
+    result = if category_id
+               resources.joins(:categories).where('categories.id' => category_id)
+             else
+               Resource.all
+             end
 
-    render json: resources.to_json(include: ['addresses', 'phones', 'categories', { schedule: { include: 'schedule_days' } }])
+    render json: result.to_json(resource_inclusion)
   end
 
   def show
-    resource = Resource.find(params[:id])
-    render json: resource.to_json(include: ['addresses', 'phones', 'categories', { schedule: { include: 'schedule_days' } }])
+    resource = resources.find(params[:id])
+    render json: resource.to_json(resource_inclusion)
+  end
+
+  private
+
+  def resources
+    Resource.includes(:addresses, :phones, :categories,
```

Comment:
@lelandg559 This should speed up the `index` action. Without this, we were making queries for each of the associated records for every rendered resource.